frame/fast-unstake/src/tests.rs

```diff
@@ -1191,10 +1080,32 @@ mod batched {
 					Event::Slashed { stash: 666, amount: Deposit::get() },
 					Event::BatchChecked { eras: vec![3] },
 					Event::Slashed { stash: 667, amount: Deposit::get() },
-					Event::BatchFinished,
+					Event::BatchFinished { size: 0 },
 					Event::BatchChecked { eras: vec![3] }
 				]
 			);
 		});
 	}
 }
+
+#[test]
+fn kusama_estimate() {
+	use crate::WeightInfo;
+	let block_time = frame_support::weights::Weight::from_ref_time(
+		frame_support::weights::constants::WEIGHT_REF_TIME_PER_SECOND * 2,
+	)
+	.ref_time() as f32;
+	let on_idle = crate::weights::SubstrateWeight::<T>::on_idle_check(1000, 64).ref_time() as f32;
+	dbg!(block_time, on_idle, on_idle / block_time);
```

Comment:
Do you want to do anything with these values?  
I see you already have a test in the Kusama runtime.

Reply:
It is kinda convenient to have them here as well because building and running it here is much faster than updating polkadot etc